src/app/beer_garden/config.py

```diff
@@ -793,7 +793,7 @@ def _parse_args(args: Sequence[str]) -> Tuple[YapconfSpec, dict]:
                     "type": "str",
                     "default": "0.0.0.0",
                     "description": "Host for the HTTP Server to bind to",
-                    "alt_env_names": ["PARENT_HOST"],
+                    "alt_env_names": ["PARENT_HTTP_HOST"],
```

Comment:
Same with this - "PARENT_HTTP_HOST" is already the environment name for this config item.